app/models/middleware_server.rb

```diff
@@ -2,5 +2,6 @@ class MiddlewareServer < ActiveRecord::Base
   include ReportableMixin
 
   belongs_to :ext_management_system, :foreign_key => "ems_id"
+  has_many :middleware_deployments, :foreign_key => "server_id"
```

Comment:
I think that you don't need to specify here the foreign key, it was already specified in the belongs_to part